pr_insight/algo/ai_handlers/litellm_ai_handler.py

```
@@ -6,7 +6,7 @@
 from litellm import acompletion
 from tenacity import retry, retry_if_exception_type, stop_after_attempt
 
-from pr_insight.algo.ai_handlers.base_ai_handler import BaseAiHandler
+from pr_insight.algo import NO_SUPPORT_TEMPERATURE_MODELS, USER_MESSAGE_ONLY_MODELS
```

Ruff (0.8.2)

`pr_insight.algo.NO_SUPPORT_TEMPERATURE_MODELS` imported but unused

(F401)